app/api/probe_dock/metrics_api.rb

```diff
@@ -117,6 +119,40 @@ def current_organization
         result
       end
 
+      get :projectHealth do
+        authorize! :organization, :data
+
+        rel = TestDescription.joins(:project_version)
+
+        # Retrieve the data from the specified version otherwise take the most recent project version to retrieve the data
+        if params[:projectVersionId].present?
+          rel = rel.joins(:project_version).where('project_versions.api_id = ?', params[:projectVersionId])
+        else
+          project_version = ProjectVersion.joins(:project).where('projects.api_id = ?', params[:projectId]).order('created_at DESC').first
+          rel = rel.where('project_versions.id = ?', project_version.id)
+        end
+
+        # Calculate the different metrics
+        result = rel.to_a.inject({testsCount: 0, passedTestsCount: 0, inactiveTestsCount: 0, inactivePassedTestsCount: 0}) do |memo, data|
+          memo[:testsCount] += 1
```

Comment:
`testsCount` should be the total number of tests, meaning you should retrieve it with a query on `ProjectTest`, otherwise it doesn't take unused tests into account.